### DISCUSSION TITLE
Backpack v6 and tabler dark theme

### DISCUSSION THREAD

**Discussion by georgedobre** [Laravel-Backpack/community-forum#531] [Q&A (Help)] [answered]
Hi guys, I've been searching around but can't seem to find it, so apologies in advanced if i missed it.
I purchased the new version 6 of backpack and i would love to start developing with the tabler horizontal dark theme, any chance on getting some pointers of how to install it?

LE: I installed the actual dev package I found on [packagist](https://packagist.org/packages/backpack/theme-tabler) but can't find any further instructions.

Thank you,
George

**Comment by christmex** [comment, marked as answer]
Hi @georgedobre maybe you can try this one if you use backpack v6

Create a new file in ``` config/backpack/theme-tabler.php ``` and inside that file, copy this
```php
<?php

return [

    /**
     * 1st layer of customization
     *
     * Simple pick a layout and let Backpack decide the best look for it.
     * No extra step is required.
     *
     * Possible values: horizontal, horizontal_dark, horizontal_overlap, vertical,
     * vertical_dark, vertical_transparent (legacy theme), right_vertical, right_vertical_dark, right_vertical_transparent
     */
    'layout' => 'horizontal_dark',

    /**
     * Pick a login page layout.
     * Possible values: default, illustration, cover
     */
    'auth_layout' => 'default', // default, illustration, cover

];
```


Let me know if you found any issue, thx

**Comment by georgedobre** [comment]
Hi @christmex sorry I haven't replied, just managed to find some time to test it. My problem at it's core was the fact that I just bought v6 as my previous licence expired and I used the instructions on the website, so when i ran php artisan backpack:version it was 5.x. So I just reinstalled it with the instructions from the docs then added the snippet you gave me, worked like a charm.

Thank you
